packages/remix-google-cloud-functions/__tests__/server-test.ts

```diff
@@ -0,0 +1,271 @@
+import supertest from "supertest";
+import { createRequest } from "node-mocks-http";
```

Comment:
I think these packages should be in `devDependencies` as well?

Reply:
@danny-yu 's added these